apps/metadata_playground/app.py

```diff
@@ -57,6 +57,9 @@ def run_steps() -> None:
 def get_data_page_url() -> str:
     """Get data page URL"""
     HOST = config.DB_HOST
+    if HOST == "127.0.0.1":
+        # The following port is defined in one of owid-grapher's config files.
+        HOST = "localhost:3030"
```

Comment:
```suggestion
        HOST = f"localhost:{config.DB_PORT}"
```

Reply:
That `DB_PORT` is a different parameter, as far as I understand. I think the actual port (3030) is not defined anywhere in ETL, but in owid-grapher.

Reply:
you are right!